modules/local/selfcomp_alignmentblocks.nf

```diff
@@ -1,6 +1,8 @@
 process SELFCOMP_ALIGNMENTBLOCKS {
     tag "$meta.id"
-    label 'process_single'
+    label 'process_medium'
+    publishDir "", enabled: false
```

Comment:
This line is not needed since you've removed the global `publishDir` option in `conf/modules.config`.

Reply:
This is retained from testing, i'll remove now.